columnphysics/icepack_therm_vertical.F90

```diff
@@ -2296,6 +2351,10 @@ subroutine icepack_step_therm1(dt,                        &
          fswthru_vdf , & ! vis dif shortwave penetrating to ocean (W/m^2)
          fswthru_idr , & ! nir dir shortwave penetrating to ocean (W/m^2)
          fswthru_idf , & ! nir dif shortwave penetrating to ocean (W/m^2)
+         fswthru_uvrdr,& ! vis uvr dir shortwave penetrating to ocean (W/m^2)
+         fswthru_uvrdf,& ! vis uvr dif shortwave penetrating to ocean (W/m^2)
+         fswthru_pardr,& ! vis par dir shortwave penetrating to ocean (W/m^2)
+         fswthru_pardf,& ! vis par dif shortwave penetrating to ocean (W/m^2)
```

Comment:
The new radiation parameters need to be explained in the documentation.  What are uvr and par?  Are they associated with particular bands?

Reply:
I have updated the documentation in the code and in the user guide.  done.